types/Keyguard.d.ts

```diff
@@ -78,7 +78,13 @@ type Parsed<T extends KeyguardRequest.Request> =
     T extends Is<T, KeyguardRequest.ImportRequest> ?
         Transform<
             KeyguardRequest.ImportRequest,
+            'isKeyLost' | 'wordsOnly',
+            { isKeyLost: boolean, wordsOnly: boolean }
+        > :
+    T extends Is<T, KeyguardRequest.ResetPasswordRequest> ?
+        Transform<
+            KeyguardRequest.ResetPasswordRequest,
             'isKeyLost' | 'expectedKeyId' | 'wordsOnly',
-            { isKeyLost: boolean, expectedKeyId?: KeyInfo, wordsOnly: boolean }
+            { isKeyLost: boolean, expectedKeyId: KeyInfo, wordsOnly: boolean }
```

Comment:
I would still prefer `expectedKeyId` to be a string.

Reply:
I simply forgot that I wanted to include that suggestion. Is a string now.